src/nova_session.erl

```diff
@@ -103,7 +103,7 @@ get_session_module() ->
             Module;
         _ ->
             %% Default to nova_session_ets
-            nova_session_cache
+            nova_cache
```

Comment:
This should be `nova_session_ets`. The nova_cache is used for other use-cases :)